kentik_api_library/tests/e2e/cloudexport/test_aws.py

```diff
@@ -42,56 +39,33 @@ def test_aws_crud() -> None:
         created = client().cloud_export.create(AWS)
         assert isinstance(created, CloudExport)
         assert created.id != ID()
-        assert created.type == CloudExportType.CUSTOMER_MANAGED
-        assert created.enabled == True
-        assert created.name == "e2e-aws-cloudexport"
-        assert created.description == "E2E test AWS CloudExport description"
-        assert created.api_root == "https://api.kentik.com"
-        assert created.flow_dest == "https://flow.kentik.com"
-        assert created.plan_id == PLAN_ID
-        assert created.cloud_provider == "aws"
-        assert created.aws == PROPERTIES1
+        assert created.type == AWS.type
+        assert created.enabled == AWS.enabled
+        assert created.name == AWS.name
+        assert created.description == AWS.description
+        assert created.plan_id == AWS.plan_id
+        assert created.cloud_provider == AWS.cloud_provider
+        assert created.aws == AWS.aws
```

Comment:
This may fail due to differences in "runtime state", which is reflected in the `_current_status` attribute. This would only happen if the suite was executed with valid "cloud-side" config and credentials and even then the state may not update fast enough. It would be though better to exclude `_current_status` from comparisons.

Reply:
OK, will clear the read-only attributes from comparison